src/node_bundler/index.js

```diff
@@ -35,6 +44,8 @@ const bundleJsFile = async function ({
     outfile: bundlePath,
     nodePaths: additionalModulePaths,
     platform: 'node',
+    plugins,
+    resolveExtensions: ['.js', '.jsx', '.mjs', '.cjs', '.json'],
```

Comment:
Should `.es6` be added? This is an old extension that was used in the past when ES6 became a thing.

Reply:
Sure! I'll make a note to do some testing and add that in a subsequent PR.